docs/user_manual/source/pipeline.rst

```diff
@@ -87,11 +83,25 @@ and zero stall on the data-side memory interface.
   |                       | 4 (target is a non-word-aligned      | EX stage and will cause a flush of the IF stage (including  |
   |                       | non-RVC instruction)                 | prefetch buffer) and ID stage.                              |
   +-----------------------+--------------------------------------+-------------------------------------------------------------+
-  | Instruction Fence     | 5                                    | The FENCE.I instruction as defined in 'Zifencei' of the     |
+  | Instruction Fence     | 5                                    | The FENCE[.I] instructions are defined in 'Zifencei' of the |
```

Comment:
I assume that FENCE[.I] in the right most column is supposed to cover both fence and fence.i. If so, just add a separate row for the fence instruction as it is not part of Zifencei as implied now.

Reply:
Fixed